face/src/components/execution/ExecutionView.js

```diff
@@ -26,6 +28,17 @@ class ExecutionView extends View {
     onStepClick(e) {
         this.$(e.currentTarget).parent().toggleClass('step_expanded');
     }
+
+    @on('click .attachment-row')
+    onAttachmentClick(e) {
+        const attachmentUid = $(e.currentTarget).data('uid');
+        router.toUrl(this.options.baseUrl + '/' + attachmentUid);
+    }
+
+    @on('click .attachment-row__link')
+    onAttachmentFileClick(e) {
+        e.stopPropagation();
```

Comment:
There is a difference between `.attachment-row` and `.attachment-row__link`. Attachment-link has `target="_blank"` and will be opened in a new tab. No need to prevent this.